test/e2e/resource_fairness/resource_fairness_test.go

```diff
@@ -24,36 +24,116 @@ import (
 	"time"
 
 	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/apache/yunikorn-core/pkg/common/configs"
+	"github.com/apache/yunikorn-core/pkg/webservice/dao"
 	"github.com/apache/yunikorn-k8shim/pkg/common/constants"
 	tests "github.com/apache/yunikorn-k8shim/test/e2e"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/common"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/k8s"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/yunikorn"
 )
 
+var kClient k8s.KubeCtl
+var restClient yunikorn.RClient
+var nodes *v1.NodeList
+var workerID1 string
+var workerID2 string
+var namespace *v1.Namespace
+var err error
+var queuePath string
+var basicUtiliz float64
+
+var maxCPU int64 = 500
+var maxMem int64 = 500
+var nodesToTaint = []string{}
+var taintKey = "e2e_test_fairness"
+var ns = "test-" + common.RandSeq(10)
+
+var _ = BeforeSuite(func() {
+	kClient = k8s.KubeCtl{}
+	Ω(kClient.SetClient()).To(BeNil())
+
+	restClient = yunikorn.RClient{}
+	Ω(restClient).NotTo(gomega.BeNil())
+
+	yunikorn.EnsureYuniKornConfigsPresent()
+
+	ginkgo.By("Port-forward the scheduler pod")
+	err = kClient.PortForwardYkSchedulerPod()
+	Ω(err).NotTo(gomega.HaveOccurred())
+
+	ginkgo.By("create development namespace")
+	namespace, err = kClient.CreateNamespace(ns, nil)
+	gomega.Ω(err).NotTo(gomega.HaveOccurred())
+	gomega.Ω(namespace.Status.Phase).To(gomega.Equal(v1.NamespaceActive))
```

Comment:
Drop this and run all tests in a separate namespace. Running multiple tests in the same namespace is asking for trouble. See https://github.com/apache/yunikorn-k8shim/pull/753 as an example.